### ISSUE TITLE
Scipy Solver parrallel solving

### ISSUE THREAD

**Issue by DaluS** [DaluS/GEMMES#121]
In today's architecture, the Rk4 homemade solve everything in parallel, making it very fast for sensitivity analysis. Scipy solver are using a loop on all systems, which is due to the way we give them the system of equation/dimensions. 

As those solvers have multiple advantages, it could be useful to get them working in parallel in the future. However as long as the RK4 homemade is working well we do not have this as a high priority !

**Comment by DaluS** [comment]
Really low priority as rk4 is working very well
